src/applications/vaos/new-appointment/components/CommunityCareProviderSelectionPage/index.jsx

```diff
@@ -80,8 +81,18 @@ function CommunityCareProviderSelectionPage({
           title="Community Care preferences"
           schema={schema}
           uiSchema={uiSchema}
-          onSubmit={() => routeToNextAppointmentPage(history, pageKey)}
-          onChange={newData => updateFormData(pageKey, uiSchema, newData)}
+          onSubmit={() => {
+            recordEvent({
```

Comment:
I think we'd want to get this covered in a unit test. Should be able to add a `Continue` button click to one of the existing tests and verify that the event is set correctly.

Reply:
Fixed